Dockerfile

```diff
@@ -1,15 +1,8 @@
-FROM mjstealey/hs_docker_base:1.7.5
+FROM mjstealey/hs_docker_base:1.9.5
 MAINTAINER Michael J. Stealey <stealey@renci.org>
 
 ### Begin - HydroShare Development Image Additions ###
-RUN pip install --upgrade pip && pip install \
-  xmltodict==0.10.2 \
-  selenium \
-  dominate \
-  django-robots
-RUN curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
-RUN apt-get update && apt-get install -y nodejs
-RUN npm install -g phantomjs-prebuilt
```

Comment:
@hyi - Those issues were related to the dockerworker not liking something and falling over. The dockerworker has already been removed due to it not being used.